core/src/test/java/org/elasticsearch/search/builder/SearchSourceBuilderTests.java

```diff
@@ -574,25 +305,4 @@ public void testEmptyQuery() throws IOException {
         String query = "{ \"query\": {} }";
         assertParseSearchSource(builder, new BytesArray(query), ParseFieldMatcher.EMPTY);
     }
-
-    public void testSearchRequestBuilderSerializationWithIndexBoost() throws Exception {
```

Comment:
Has this test moved somewhere or was it deleted for any particular reason? Just had a quick look but can't find it.